app/components/Editor/index.js

```diff
@@ -16,6 +16,7 @@ import importer from './importer';
 import exporter from './exporter';
 import { ImageButton } from './Sides';
 import { ImageBlock } from './Blocks';
+import { InfoButton } from './Sides';
```

Comment:
This can be combined with the one on line 17
`import { ImageButton, InfoButton }  from './Sides';`